### PR TITLE
SW-6066 Only update pre-screen applicaitons folders and do not update participant folders

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2462** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2462?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2468** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2468?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/main/kotlin/com/terraformation/backend/accelerator/GoogleFolderUpdater.kt [hunk 1]
```diff
@@ -25,23 +25,15 @@ class GoogleFolderUpdater(
       val application = applicationStore.fetchOneById(event.applicationId)
       val projectDetails = projectAcceleratorDetailsStore.fetchOneById(application.projectId)
 
-      // If a project has a Google folder, but doesn't have a file naming set up
-      if (projectDetails.googleFolderUrl != null && projectDetails.fileNaming == null) {
+      // If a project has a Google folder, and is still in pre-screen
+      if (projectDetails.googleFolderUrl != null &&
```
Comment (constanzauanini) [inline src/main/kotlin/com/terraformation/backend/accelerator/GoogleFolderUpdater.kt:29]: Is it correct to update it with no fileNaming?

**Review by constanzauanini — APPROVED** [review]
One question, but lgtm!

File: src/main/kotlin/com/terraformation/backend/accelerator/GoogleFolderUpdater.kt (hunk 1 above)
Comment (tommylau523) [reply src/main/kotlin/com/terraformation/backend/accelerator/GoogleFolderUpdater.kt:29]: Good question! The latest decision is to ignore file naming during pre-screen. Admins will rename the folders in GDrive directly.

**Pull request merged**